builtin/logical/nomad/client.go

```diff
@@ -0,0 +1,28 @@
+package nomad
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/vault/logical"
+)
+
+func client(s logical.Storage) (*api.Client, error, error) {
```

Comment:
Can you attach this client method to the backend?

Reply:
Solved in f3aaacc3fc687dc08fb8710cd9baf44ccf9a9ee4